P/PCRE2/build_tarballs.jl

```diff
@@ -29,6 +29,7 @@ fi
 export CFLAGS="${CFLAGS} -O3"
 
 ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
+    --disable-symvers \
```

Comment:
This looks good to me, but I'm not seeing a difference with previous builds, so it isn't clear to me whether versioned symbols were added in this version only or what.

Reply:
https://github.com/PCRE2Project/pcre2/releases/tag/pcre2-10.47
> (Build change) There are now linker scripts to enable symbol versioning for the PCRE2 dynamic libraries. Downstream Linux distributions may make use of this, or disable it with the new Autoconf `--disable-symvers` and CMake `-DPCRE2_SYMVERS` options. Linux, Solaris, and FreeBSD (GNU ld, LLVM lld, and Solaris ld) are tested and supported.

Ah, ok, if I understand correctly versioned symbols are new, and we're disabling them right away, so we don't see ABI changes between builds and all is good, yay!